Plugins/Flow.Launcher.Plugin.Explorer/Languages/en.xaml

```diff
@@ -26,7 +26,7 @@
     <system:String x:Key="plugin_explorer_edit">Edit</system:String>
     <system:String x:Key="plugin_explorer_add">Add</system:String>
     <system:String x:Key="plugin_explorer_generalsetting_header">General Setting</system:String>
-    <system:String x:Key="plugin_explorer_manageactionkeywords_header">Customise Action Keywords</system:String>
+    <system:String x:Key="plugin_explorer_manageactionkeywords_header">Customize Action Keywords</system:String>
```

Comment:
Don't know if we want to make this change, it will reset translations I believe.

Reply:
OK. Reverted.